### DISCUSSION TITLE
http/http-dns

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#141] [General]
# http/http-dns

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/http/http-dns.html

**Comment by zhaohong-ponyai** [comment]
内网环境，应该不需要这个吧？

**Reply by isno** [reply]
内网不需要，这是我做海外业务时的总结。

**Reply by zhaohong-ponyai** [reply]
麻烦请教一下，有没有企业内网的相关DNS架构图哈，比如全国有3-4个区域，每个区域有1-2个办公室，然后有很多内网服务需要域名解析，那么怎样规划比较好啊？

**Reply by isno** [reply]
没有呢，你需要自己解决了。

**Reply by zhaohong-ponyai** [reply]
好吧，谢谢了

**Comment by hitolz** [comment]
但作为一个“中间商”，“域名解析器”很容器出现域名劫持、解析时间过长、解析调度不精准等问题。

**Reply by hitolz** [reply]
能明改善传统域名解析带来的各类问题

**Reply by isno** [reply]
o(╥﹏╥)o，哎呀。我这一篇文章里面四五处错误。。。

**Comment by 113998274** [comment]
提个小问题，微信企微这些也是使用了 HTTPDNS 吗，平时用科学上网没有见这些软件有影响

**Reply by isno** [reply]
HTTPDNS 只是用来处理域名解析的，跟科学上网没什么关系

**Reply by data-agent-ts** [reply]
不受影响是因为  科学上网的客户端 有一系列规则 不会让你的微信 走代理  你开全局代理 还是有概率 出问题的

**Comment by adhope** [comment]
系统缓存域名解析应该比每次都访问域名解析服务器要快吧？
